zh_CN/messages.json

```
@@ -1717,12 +1717,33 @@
     "severity_9": {
         "message": "9"
     },
+    "Userscript_Search": {
+        "message": "用户脚本搜索"
+    },
+    "Userscript_search_integration_mode": {
+        "message": "用户脚本搜索集成"
+    },
+    "Scripts_activated_by_context_menu": {
+        "message": "使用 @run-at 上下文菜单的用户脚本"
```

你说的你以为发起者可以改可能是因为用了像下面一样的 request change
```suggestion
        "message": "使用 @run-at 上下文菜单的用户脚本"
```

---

不过没有什么大问题，等着被merge就可以了

---

😉